Master

pratikcws opened the pull request:
In all link field when we filter records via code
if you type something that doesn't exist , the field will automatically be cleared
but if you type something that exist in database but not available in filter record , just not allowed to select, it would not be cleared

Example, if i knew the full name from another company "All Warehouses - Do"
I can type it on the warehouse field, and the system doesn't clear the field out when the input field loose focus..
so we change code  
/home/frappe/frappe-bench/apps/frappe/frappe/public/js/frappe/form/controls/link.js

[link.zip](https://github.com/frappe/erpnext/files/3017039/link.zip)

on line 167 : 
if(r.results.length==0){
     frappe.model.set_value(me.df.parent, me.df.name, me.df.fieldname, "", me.df.fieldtype);
    refresh_field(me.df.fieldname);
}

nabinhait committed:
fix(uom): Get items as per UOM defined in BOM, fixes #15081

Zlash65 committed:
feat: selecting parent_company should disable chart_of_accounts based fields

Zlash65 committed:
feat: once company is saved, parent_company cannot be selected

Zlash65 committed:
feat: allow adding account only if topmost parent company
- hidden filter for Parent Company added
- Add button overriden with new condition

Zlash65 committed:
fix: override primary action button, change filter to root company

Zlash65 committed:
fix: added validation to if account is being added to child company

Zlash65 committed:
feat: sync account created for a group company to all its descendants

Zlash65 committed:
fix: set chart of accounts based on parent company on server side

Zlash65 committed:
fix: return if no descendants found

Zlash65 committed:
fix: chart of accounts field toggling improv

Zlash65 committed:
test case added to check account syncing

Zlash65 committed:
codacy fixes

rohitwaghchaure committed:
fix: e-invoicing patch, patch to set country code in the address

crossxcell99 committed:
fix: call base clase onload method

rmehta committed:
Merge pull request #16790 from crossxcell99/master

fix: call base class onload method

Anurag810 committed:
refactor

rohitwaghchaure committed:
fix: bom cost update is not working

rohitwaghchaure committed:
fix: test case

rohitwaghchaure committed:
Merge pull request #16797 from rohitwaghchaure/cost_update_not_working_for_price_list

fix: bom cost update is not working

rohitwaghchaure committed:
Merge pull request #16787 from rohitwaghchaure/italian_more_fixes

fix: e-invoicing patch, patch to set country code in the address

rohitwaghchaure committed:
fix: scan barcode not adding the barcode value in the items table

frappe-bot committed:
bumped to version 11.1.12

frappe-bot committed:
Merge branch 'hotfix'

rohitwaghchaure committed:
Removed validation for group by voucher in general ledger

finbyztech committed:
fix: Rounding adjustment in Rounded Total

In Purchase Receipt and GST Purchase Receipt report, the Rounded Total column doesn't display rounding adjustment of Base Grand Total. So, we have created a fix for that rounding adjustment.

Commit:
fix(regional,italy): bank details, custom button on sales invoice

Updates to Bank details in e-invoice XML
Additional fields on Payment Schedule
Button to Generate E-Invoice on Sales Invoice (generates and replaces attached e-invoice)
Replaced `x.decode('utf-8')` with `frappe.safe_decode(x, encoding='utf-8')` in setup.py
Updated italian localisation patch line in patches.txt

Commit:
fix(regional,italy): codacy fixes in regional sales_invoice js

rohitwaghchaure committed:
Merge pull request #16807 from rohitwaghchaure/remove_validation_for_group_by_voucher_consolidate

fix: when General Ledger is opened from Purchase Receipt or Delivery note, user get Opening and Closing printed twice

rohitwaghchaure committed:
Merge pull request #16817 from gaurav-naik/italy-einvoice-bank-details-fix

fix(regional,italy): bank details, custom button on sales invoice

frappe-bot committed:
bumped to version 11.1.13

frappe-bot committed:
Merge branch 'hotfix'

rohitwaghchaure committed:
fix: on save state code field become blank

jay-parikh committed:
Fix Sales Invoice Return Validation "validate_pos"

rohitwaghchaure committed:
fix(sales order item): not able to search item by description

Anurag810 committed:
minor fixes

Anurag810 committed:
Minor fixes

deepeshgarg007 committed:
revert: Address and contact report fix (#16786)

Reverts #16674 

Address and other info are not visible in address and contact report
![screenshot 2019-02-27 at 11 31 38 am](https://user-images.githubusercontent.com/42651287/53469309-6029f500-3a83-11e9-9672-e5e7d14dc470.png)

deepeshgarg007 committed:
fix: Print letter head only if checked in Print Settings (#16789)

Letter head is printed in financial statements even if option is not checked

![letter head option](https://user-images.githubusercontent.com/42651287/53430759-26baa080-3a15-11e9-948a-bc95700a18d2.png)

deepeshgarg007 committed:
fix: Selling and buying amount precision fix (#16764)

rohitwaghchaure committed:
fix: HSN code not disaplying for the GST Purchase Invoice print format

Anurag810 committed:
fix: gross depends upon now report type

Anurag810 committed:
Minor fixes

rohitwaghchaure committed:
Merge pull request #16629 from Anurag810/gross-and-net-profit-report

feat: Gross and Net Profit Report

rohitwaghchaure committed:
Merge pull request #16704 from Anurag810/vedmata-print-formats

feat: Auditors print formats

rohitwaghchaure committed:
Merge pull request #16829 from rohitwaghchaure/gst_purchase_invoice_hsn_number

fix: HSN code not disaplying for the GST Purchase Invoice print format

rohitwaghchaure committed:
Merge pull request #16824 from rohitwaghchaure/user_not_able_to_search_item_by_description

fix(sales order item): not able to search item by description

rohitwaghchaure committed:
Merge pull request #16820 from rohitwaghchaure/state_code_blank_if_not_matched_with_state_codes

fix: on save of the address, state code field become blank

rmehta committed:
Merge pull request #16823 from jay-parikh/sales-return

fix: Python 3 Sales Invoice Return Validation "validate_pos"

hrwX committed:
Allow zero amount in additional salary

pratu16x7 committed:
Merge pull request #16842 from hrwX/additional_salary

fix(Additional Salary): Allow additional salary to be zero

barredterra committed:
fix(projects): change fieldtype of billing amounts to Currency

rmehta committed:
fix(sales_order.py): handle zero bundle qty

jay-parikh committed:
Fix Company Default to Total Stock Summary Report

rohitwaghchaure committed:
POS profile, account for change amount must be cash or bank account

hrwX committed:
sales_order to work_order item desc fix

hrwX committed:
fix sales order test for creating work order from sales order

hrwX committed:
[debug] locally tests passed but travis failed

hrwX committed:
remove debug code

rohitwaghchaure committed:
Merge pull request #16853 from rohitwaghchaure/pos_profile_changes

fix: POS profile, account for change amount must be cash or bank account

rohitwaghchaure committed:
Merge pull request #16806 from rohitwaghchaure/set_scanned_barcode_in_the_table

fix: scan barcode not adding the barcode value in the items table

rohitwaghchaure committed:
fix: added tags for discount amount, multiple customer po no, delivery notes

Anurag810 committed:
fix: pass correct argument to method get_deprication_amount

rohitwaghchaure committed:
feat: added provision to disable CWIP accounting in asset settings

deepeshgarg007 committed:
fix: Remove validation for relieving date from additional salary

nabinhait committed:
Update item_barcode_childtable_migrate.py

rohitwaghchaure committed:
fix: purchase receipt not able to submit because default inventory account has not selected in another company

frappe-bot committed:
bumped to version 11.1.14

frappe-bot committed:
Merge branch 'hotfix'

hrwX committed:
fetch from fix

rohitwaghchaure committed:
feat: child table to add multiple time logs in job card

Anurag810 committed:
feat: timesheet Employee Summary Report

Anurag810 committed:
Commonify code

Anurag810 committed:
Minor Fixes

Anurag810 committed:
feat: Project Billing Summary for timesheet

Anurag810 committed:
fix: readability

Anurag810 committed:
Added mandatory feilds to the report

Anurag810 committed:
fix: Removed raw query and used frappe.get_all

rohitwaghchaure committed:
fix: ImponibileImporto calculation

rmehta committed:
Merge pull request #16866 from deepeshgarg007/additional

fix: Removed relieving date validation form additional salary

rmehta committed:
Merge pull request #16856 from hrwX/sales-work-fix

fix(Work Order): fetch item info when making work order from sales order

rmehta committed:
Merge pull request #16854 from jay-parikh/hotfix

fix: Default Company to Total Stock Summary Report

rmehta committed:
Merge pull request #16892 from hrwX/uae_fix

fix(UAE): Fetch from fix for UAE customizations

elba7r committed:
fix: add returns field to cashier closing  (#16911)

rmehta committed:
Update payment_request.py

rohitwaghchaure committed:
Merge pull request #16882 from rohitwaghchaure/multiple_time_logs_in_job_card

feat: child table to add multiple time logs in job card

rushin29 committed:
fix: gst_state_number for address with Unregistered GST (#16798)

rohitwaghchaure committed:
Merge pull request #16865 from rohitwaghchaure/purchase_invoice_asset_issue_not_able_to_submit

feat: added provision to disable CWIP accounting in asset settings

deepeshgarg007 committed:
fix: Gross profit report fix (#16935)

deepeshgarg007 committed:
fix: Change IBAN Account length from 25 to 30 (#16847)

rohitwaghchaure committed:
fix: test cases and codacy

hrwX committed:
disbale pl check for Sales and Purchase Invoice

rohitwaghchaure committed:
Merge pull request #16523 from Zlash65/comp-sync

feat: Sync Chart of Accounts in Company hierarchy

hrwX committed:
fix(Customer Ledger): ambiguous error in where clause (#16914)

fix for error "InternalError: (1052, u"Column 'company' in where clause is ambiguous")" in Customer Ledger Summary

rohitwaghchaure committed:
Merge pull request #16942 from hrwX/invoice-creation-tool

fix(Invoice Creation Tool): Disable P/L validation for opening invoice creation

hrwX committed:
refactor: fix issue credit note

hrwX committed:
fix: change customer to supplier

rmehta committed:
Merge pull request #16951 from hrwX/change-to-supplier

fix(Supplier Ledger Summary): Change Customer to Supplier in Supplier Ledger Summary

rohitwaghchaure committed:
fix: advance amount is not calculating properly

rohitwaghchaure committed:
Merge pull request #16952 from rohitwaghchaure/fixed_accounts_receivable_summary_advance_amt

fix: advance amount is not calculating properly

rohitwaghchaure committed:
fix: POS, cost center not added against debtors

rohitwaghchaure committed:
Merge pull request #16954 from rohitwaghchaure/pos_cost_center_not_added_against_debtors

fix: POS, cost center not added against debtors

rohitwaghchaure committed:
fix: Accounts receivable report, remarks showing in the sales person column

Mangesh-Khairnar committed:
fix: handle an empty match query condition in leave application (#16931)

rohitwaghchaure committed:
Merge pull request #16955 from rohitwaghchaure/sales_person_showing_wrong_on_ars

fix: Accounts receivable report, remarks showing in the sales person column

rohitwaghchaure committed:
fix: Cost center from PO to PI is not mapping

rohitwaghchaure committed:
Merge pull request #16964 from rohitwaghchaure/fix_cost_center_mapping_not_working

fix: Cost center from PO to PI is not mapping

saurabh6790 committed:
bumped to version 11.1.15

saurabh6790 committed:
Merge branch 'hotfix'

chdecultot committed:
Fix: Mapping issue in sql query

rmehta committed:
Merge pull request #16980 from chdecultot/fec_correctioon

fix: [Regional] [FEC report] Mapping issue in sql query

rohitwaghchaure committed:
feat: enabled allow on submit for action fields in the Budget

rmehta committed:
Merge pull request #16861 from Anurag810/asset_fixes

fix: passing the correct argument in get_depreciation_amount

rmehta committed:
Merge pull request #16749 from Alchez/hotfix-cancelled-payments

fix(payments): Only check for Draft or Submitted payments while cancelling payment requests

rmehta committed:
Merge pull request #16848 from alyf-de/hotfix-1

fix(projects): change fieldtype of billing amounts to Currency

rmehta committed:
Merge pull request #16869 from rohitwaghchaure/not_able_to_submit_purchase_receipt

fix: purchase receipt not able to submit because default inventory account has not selected in another company

rmehta committed:
Merge pull request #16816 from finbyz/hotfix

fix: Rounding adjustment in Rounded Total

surajshetty3416 committed:
fix: Show amount of entry based on diff of credit and debit

rmehta committed:
fix: remove gitter chat

rmehta committed:
fix(link): help link in selling

pratu16x7 committed:
fix: correct use of include_item_in_manufacturing

rohitwaghchaure committed:
Merge pull request #16983 from rohitwaghchaure/allow_on_submit_for_budget_fields

feat: enabled allow on submit for action fields in the Budget

hrwX committed:
fix: include fiscal code in invoice xml if present

SaiFi0102 committed:
fix(Party Ledger Summary): Show all adjustment entries as columns
Consider only Income Account or Expense Account account type for adjustment amounts
Check if GL Entry's is_opening == "Yes"

hrwX committed:
CodiceFiscale should be after IdFiscaleIVA

frappe-bot committed:
bumped to version 11.1.16

frappe-bot committed:
Merge branch 'hotfix'

hrwX committed:
fix: Cost center as per company

rohitwaghchaure committed:
fix: description missing in the XML (#16985)

scmmishra committed:
fix: Task Overdue status propagates to project

- Used ORM instead of a SQL for set_task_as_overdue
- update_status function compares the time and marks the appropriate task as overdue
- The on_update hook makes the changes in project as well

scmmishra committed:
feat: Added check for no end date and refactored query

scmmishra committed:
fix: Typo in update_status

scmmishra committed:
fix: minor fix in test

deepeshgarg007 committed:
fix: Child company account currency fix

rmehta committed:
Merge pull request #16986 from hrwX/italy_xml_fix

fix(Italy Customisation): Include fiscal code in invoice xml if present

rohitwaghchaure committed:
Merge pull request #17004 from surajshetty3416/fix-bank-reconciliation-entries-hotfix

fix: Show consolidated amount of payment entries in Bank reconciliation

rohitwaghchaure committed:
fix: incorrect paid amount in accounts receivable summary report

rohitwaghchaure committed:
Merge pull request #17011 from rohitwaghchaure/fix_incorrect_paid_amt_in_accounts_receivable_summary_report

fix: incorrect paid amount in accounts receivable summary report

netchampfaris committed:
fix: Reorder Currency column to fix print (#17008)

Every Financial Report's 2nd column is Currency, which is an implicit requirement to Print them properly. In this case, Income column was not printed. This fixes that.

deepeshgarg007 committed:
fix: Added setting in company to ignore root company validation

barredterra committed:
fix(projects): Auto-set employee and company in Timesheet (#16743)

* fix(projects): auto-set employee and company

* only set employee and company if document is unsaved

* use frm.set_value()

deepeshgarg007 committed:
fix: Allow to add child account if ignore validation checked

deepeshgarg007 committed:
fix: Missing semicolon

deepeshgarg007 committed:
fix: Change name to Allow Account Creation against Child Company in company master

nabinhait committed:
Merge pull request #17006 from deepeshgarg007/account-fix

fix: Ignore root company validation setting in company master

pratu16x7 committed:
Merge pull request #17019 from pratu16x7/hotfix-stock_entry_from_bom

fix(uom): Get items as per UOM defined in BOM

rohitwaghchaure committed:
fix: offline pos, duplicate records creating

hrwX committed:
fix(Leave Balance Report): total allocated leaves not calculated properly (#16969)

* fix: total alloted leaves not calculated properly

* fix: Possible SQL injection

* typo fix in sql query

* prevent sql injection again ?

* Use ORM for query

rohitwaghchaure committed:
Merge pull request #17020 from rohitwaghchaure/offline_pos_duplicate_records_making

fix: offline pos, duplicate records creating

scmmishra committed:
feat: refactored update status

scmmishra committed:
Merge branch 'hotfix-task-status' of github.com:scmmishra/erpnext into hotfix-task-status

sagarvora committed:
fix(accounts): dependent field should be displayed after source link field

surajshetty3416 committed:
fix: Commonify get gl_entry code by moving it to accounts_controller

surajshetty3416 committed:
style: Fix formatting

Commit:
fix(regional,italy): autoset tax id, cf from customer before validating missing values

Fixes issue #17033

saurabh6790 committed:
Merge pull request #16859 from rohitwaghchaure/added_new_tags_e_invoice_xml

fix: added tags for discount amount, multiple customer po no, delivery notes

Commit:
fix(regional,italy): autoset company cf, tax_id before validating

rohitwaghchaure committed:
Merge pull request #17032 from surajshetty3416/fix-group-same-items-in-printview

fix(print): Fix group same items for print view

saurabh6790 committed:
Merge pull request #17035 from gaurav-naik/issue-17033

fix(regional,italy): autoset tax id, cf from customer before validating

saurabh6790 committed:
Merge branch 'hotfix'

saurabh6790 committed:
bumped to version 11.1.17

surajshetty3416 committed:
fix: Subquery in get_bom_items method

netchampfaris committed:
fix: Missing "and" in filter conditions

nabinhait committed:
Refactored deferred revenue booking logic

nabinhait committed:
fix(accounts): codacy fixes

rohitwaghchaure committed:
fix: bank reconciliation not working

rohitwaghchaure committed:
fix: accounts receivable summary

rohitwaghchaure committed:
Merge pull request #17048 from rohitwaghchaure/fixed_bank_reconciliation_not_working

fix: bank reconciliation not working

rohitwaghchaure committed:
Merge pull request #17051 from rohitwaghchaure/accounts_receivable_summary_issue

fix: accounts receivable summary

netchampfaris commented:
@pratikcws Hey, you should open this PR against the `hotfix` branch, also rebase your branch with `hotfix`.

nabinhait committed:
Merge pull request #17046 from nabinhait/deferred_revenue

Refactor: deferred revenue booking logic

nabinhait committed:
fix(accounts): Indentation

nabinhait committed:
Merge branch 'deferred_revenue' into hotfix

rmehta committed:
fix(test): item variant

netchampfaris committed:
Merge branch 'hotfix' of https://github.com/frappe/erpnext into fix-leave-application-condition

deepeshgarg007 committed:
fix: fetch account only if available

hrwX committed:
fix: get credit and debit in float and not string

rohitwaghchaure committed:
Merge pull request #17058 from hrwX/bank_reco_fix

fix(Bank Reconciliation): get credit and debit in float and not string

rohitwaghchaure committed:
Merge pull request #17000 from scmmishra/hotfix-task-status

fix: Task overdue status propagates to project

hrwX committed:
fix(Salary Slip): Consider Leave without Pay for calculation (#16926)

while net salary calculation, it was not considering actual days worked but now it will consider it

scmmishra committed:
fix: healtcare unit type enable toggle

rohitwaghchaure committed:
fix: user not able to create chart of account in parent if child has no chart of account

rohitwaghchaure committed:
Merge pull request #17059 from rohitwaghchaure/cherry_pick_test_case_fix

fix: cherry-pick item variant test cases fix

scmmishra committed:
Merge branch 'hotfix' of https://github.com/frappe/erpnext into hotfix

rohitwaghchaure committed:
Merge pull request #17063 from scmmishra/hotfix

fix: healthcare unit type enable toggle

rohitwaghchaure committed:
Merge pull request #17064 from rohitwaghchaure/not_able_to_create_chart_of_account

fix: user not able to create chart of account in parent if child has no chart of account

hrwX committed:
fix(Update GSTIN): remove '.__' from template (#17061)

'.__'  Avoid Server Side Template Injection

rohitwaghchaure committed:
Merge pull request #16950 from hrwX/fix-issue-credit

Fix(Delivery Note): Fix issue credit note

rohitwaghchaure committed:
Revert "Fix(Delivery Note): Fix issue credit note"

rohitwaghchaure committed:
Merge pull request #17039 from surajshetty3416/fix-bom-item-subquery-issue

fix: Subquery in get_bom_items method

rohitwaghchaure committed:
Merge pull request #17069 from frappe/revert-16950-fix-issue-credit

feat: Revert "Fix(Delivery Note): Fix issue credit note"

saurabh6790 committed:
Merge branch 'hotfix'

saurabh6790 committed:
bumped to version 11.1.18

Commit:
fixed spelling in validation message

fixed typo in auto-reorder validation message

nabinhait committed:
fix: woocommerce settings

nabinhait committed:
Merge pull request #17074 from nabinhait/woocommerce-fix

fix: Woo-commerce fix

deepeshgarg007 committed:
fix: Allow doctype only in party account types

adityahase committed:
fix: Raise exception if apps are on incompatible branches

nabinhait committed:
Merge pull request #17055 from deepeshgarg007/tds

fix: TDS payable monthly report fix

nabinhait committed:
Merge pull request #17038 from netchampfaris/fix-leave-application-condition

fix: Missing "and" in filter conditions

deepeshgarg007 committed:
Merge branch 'hotfix' of https://github.com/frappe/erpnext into payment_entry

nabinhait committed:
fix: Fetch HSN code only if empty

nabinhait committed:
fix: Show Sales Person in AR/AP print is optional

rohitwaghchaure committed:
fix: calcualtion of ImponibileImporto if tax rate is zero

nabinhait committed:
Merge pull request #17088 from nabinhait/hsn-fixes

fix: Fetch HSN code only if empty

nabinhait committed:
fix: Validate returned qty with ref qty considering field precision

nabinhait committed:
fix: Validate gl entry debit/credit as per field's precision

hrwX committed:
fix(India): Add HSN/SAC in setup

rohitwaghchaure committed:
Merge pull request #17095 from rohitwaghchaure/incorrect_calculation_of_imponibile_importo

fix: incorrect calcualtion of ImponibileImporto if tax rate is zero

PawanMeh committed:
[fix] #16894

deepeshgarg007 committed:
fix: Divide by zero exception handling

nabinhait committed:
fix: Validate gl entry debit/credit as per field's precision

rohitwaghchaure committed:
fix: incorrect PrezzoTotale

rohitwaghchaure committed:
Merge pull request #17118 from rohitwaghchaure/_incorrect_prezzototale

fix: incorrect PrezzoTotale

asbasawaraj committed:
Removed Location from Abbreviation Update DocTypes

rmehta committed:
Merge pull request #17109 from PawanMeh/fixes_16894_v11

fix: Amazon v11 integration issues

rmehta committed:
fix(kaizen): warehouse account description

nabinhait committed:
Merge pull request #17113 from deepeshgarg007/item-register

fix: Divide by zero exception handling

nabinhait committed:
Merge pull request #17126 from rmehta/warehouse-account-description

fix(kaizen): warehouse account description

nabinhait committed:
Merge pull request #17100 from nabinhait/gl-entry-precision

fix: Validate gl entry debit/credit as per field's precision

nabinhait committed:
Merge pull request #17099 from nabinhait/return-precision-issue

fix: Validate returned qty with ref qty considering field precision

hrwX committed:
fix: remove duplicate doctype

nabinhait committed:
Merge pull request #17101 from hrwX/india_setup

fix(India): Add HSN/SAC in setup

nabinhait committed:
Merge pull request #17092 from nabinhait/ar-ap-print

fix: Show Sales Person in AR/AP print is optional

nabinhait committed:
Merge pull request #17086 from adityahase/check-version-hf

fix: Raise exception if apps are on incompatible branches

nabinhait committed:
Merge pull request #17087 from deepeshgarg007/payment_entry

Party type validation in payment entry

nabinhait committed:
Update item.py

nabinhait committed:
Merge pull request #17072 from kennethsequeira/auto-re-val

fix: spelling in validation message for Auto-reorder

nabinhait committed:
Merge pull request #16992 from hrwX/cost_center

fix(Purchase Invoice): Cost Centre as per company

nabinhait committed:
Merge pull request #17028 from sagarvora/field_order

fix(accounts): dependent field should be displayed after source link field

nabinhait committed:
Merge pull request #16987 from SaiFi0102/Party-Ledger-Summary-Fix

fix(Party Ledger Summary): Show all adjustment entries as columns

nabinhait committed:
Merge pull request #17122 from frappe/asbasawaraj-remove-location-from-abbreviation-doctypes

fix: Removed Location from Abbreviation Update DocTypes

hrwX committed:
fix: check transactions for batch, serial check

hrwX committed:
pull from hotfix

netchampfaris committed:
fix(Item Group): Don't clean description html

hrwX committed:
fix: get bill_no

hrwX committed:
refactor: declare bill_on at function start

rohitwaghchaure committed:
fix: incorrect POS profile selected

rohitwaghchaure committed:
always set the company from the POS Profile

nabinhait committed:
fix: Default variant Based on

rohitwaghchaure committed:
Merge pull request #17141 from rohitwaghchaure/incorrect_pos_profile_selected_if

fix: incorrect auto POS profile selected in the sales invoice

nabinhait committed:
Merge pull request #17144 from nabinhait/variant-fix

fix: Default variant Based on

nabinhait committed:
Merge pull request #17137 from netchampfaris/item-group-clean-description

fix(Item Group): Don't clean description html

nabinhait committed:
Merge branch 'hotfix' into validate_stock_transaction_v11

nabinhait committed:
Merge pull request #17133 from hrwX/validate_stock_transaction_v11

fix(Item): check transactions for batch, serial check

nabinhait committed:
Merge pull request #17139 from hrwX/pe_supp_inv_v11

fix(Payment Entry): Return Bill No for Payment Entry

saurabh6790 committed:
bumped to version 11.1.19

saurabh6790 committed:
Merge branch 'hotfix'

nabinhait committed:
Merge pull request #17146 from Anurag810/timesheet-report

feat: Timesheet report

nabinhait committed:
fix: unique variant based on

saurabh6790 committed:
bumped to version 11.1.20

saurabh6790 committed:
Merge branch 'hotfix'

nabinhait commented:
@pratikcws The branches are not correct in your PR. Please fix it and reopen a new PR.
